packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setConfig.ts

```diff
@@ -52,6 +129,9 @@ const makeContractInput = async (input: SetConfigInput): Promise<ContractInput>
   return [oracles, new BN(input.f).toNumber(), onchainConfig, 2, bytesToFelts(offchainConfig)]
 }
 
+// TODO: beforeExecute attempt to deserialize offchainConfig to check for validity
+// TODO: diff onchain config vs proposed config
```

Comment:
Is it planned to be addressed in this PR?

Reply:
todo is captured in https://smartcontract-it.atlassian.net/browse/BCI-1379 (separate PR)